cmake/lint.cmake

```
@@ -31,15 +31,26 @@ set(AUTOGRAPH_LINT_PATHS
         ${PROJECT_SOURCE_DIR}/dali/test/python/autograph/
 )
 
+add_custom_target(lint-python-black
+        COMMAND
+          black --check --config ${PROJECT_SOURCE_DIR}/pyproject.toml ${PYTHON_LINT_PATHS} ${AUTOGRAPH_LINT_PATHS}
+        COMMENT
+          "Performing black Python formatting check"
+)
 
-add_custom_target(lint-python
+
+add_custom_target(lint-python-flake
         COMMAND
           flake8 --config=${PROJECT_SOURCE_DIR}/.flake8 ${PYTHON_LINT_PATHS}
         COMMAND
           flake8 --config=${PROJECT_SOURCE_DIR}/.flake8.ag ${AUTOGRAPH_LINT_PATHS}
         COMMENT
           "Performing Python linter check"
 )
+add_dependencies(lint-python-flake lint-python-black)
+
+add_custom_target(lint-python)
+add_dependencies(lint-python lint-python-flake)
```

Why adding flake dependency from black? Would it be better to keep them separate?
```suggestion
add_custom_target(lint-python)
add_dependencies(lint-python lint-python-flake lint-python-black)
```

---

My intention is to check the linter only after the code is already formatted. Otherwise it is a moving target.